designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessRepository.scala

```diff
@@ -28,12 +28,20 @@ import scala.language.higherKinds
 
 object ProcessRepository {
 
+  private def forwardedNameWrapper(name: String) = s"Remote[$name]"
```

Comment:
Instead of having this method here, IMO better option would be to pass everywhere new value class `RemoteUserName` instead of `String` and in the last possible moment (https://github.com/TouK/nussknacker/pull/4294/files#diff-c41b00a7511457085374bebee61f84f69583d44160dc9fae32f44661148f58ffR78) do `action.forwardedUserName.map(_.display).getOrElse(loggedUser.username)`

Reply:
That is a good idea - I've added new value class :+1: